pages/index.js

```diff
@@ -1,20 +1,19 @@
 import React from 'react';
 import Layout from '../components/Layout';
 import EventList from '../components/EventList';
-import EVENTS from '../fixtures/events'
+import { fixtureApi } from '../services';
 
 export default class Index extends React.Component {
-  static async getInitialProps () {
-    const res = await fetch('https://jsonplaceholder.typicode.com/posts');
-    const json = await res.json();
-    return { json: json, events: EVENTS }
+  static async getInitialProps() {
+    const events = await fixtureApi.getAllEvents();
+
+    return { events: events._embedded['osdi:events'] };
```

Comment:
Great. I would maybe bring this out into `transforms/index.js` file to include build the structure we want along with the funky uuid.